src/Chainweb/Pact/RestAPI.hs

```diff
@@ -128,6 +136,26 @@ pactSpvApi
     . Proxy (PactSpvApi v c)
 pactSpvApi = Proxy
 
+-- -------------------------------------------------------------------------- --
+-- GET Pact SPV Proof
+
+--  | The chain endpoint is the target chain of the proof, i.e. the chain where
+--  the root of the proof is located.
+--
+type PactSpv2Api_
+    = "pact"
+    :> "spv2"
```

Comment:
Can we consider a different name here? It will be hard to justify to users why it's just called `spv2`, especially since that sounds like an update to the first version.

Reply:
Unless this _is_ actually the v2 (ie, v1 with more hash functions?)

Reply:
This new SPV API should still be considered experimental. The goal is, that once it is finalized, we get a unified SPV proof format that can be used with bridges as well as internally for Chainweb.

The new SPV API is meant to be a generalization that subsumes and replaces the previous API. However, additional work has to be done such that output can be used with pact cross chain transactions.